lib/gds-sso/config.rb

```diff
@@ -18,6 +20,9 @@ module Config
       mattr_accessor :auth_valid_for
       @@auth_valid_for = 20 * 3600
 
+      mattr_accessor :cache
+      @@cache = ActiveSupport::Cache::NullStore.new
```

Comment:
I'm in two minds about whether this should default to `MemoryStore`. At least then there's some caching by default?